### ISSUE TITLE
[A11y]Focus is provided for non interactive section of dialog, tabbing further from the end button focus does not go to 'Close' button

### ISSUE THREAD

**Issue by qjkobe** [MicrosoftDocs/msteams-docs#11387]
### Steps to reproduce

Take jira cloud app as example as it has a similar UI with our app.
Focus is provided for non interactive section of dialog. That is the DOM of whole iframe content.
Please refer to the video.

https://github.com/user-attachments/assets/8c036e31-ba3e-4da0-8805-ac96077324c2

### Expected behavior

When focusing the last button, The focus should turn to close button of the Task Module when pressing "Tab".

### Actual behavior

Focus is provided for non interactive section of dialog.

**Comment by Meghana-MSFT** [comment]
@qjkobe - We checked this at our end, and we are able to move the focus to close button by pressing Shift+Tab keys. Could you please check if this works for you?

**Comment by qjkobe** [comment]
Hi @Meghana-MSFT , 
This works for me. Is this worked as Designed in Teams App? Hope you can confirm that. Then I will reply it to our A11y Team.

**Comment by Meghana-MSFT** [comment]
@qjkobe - Yes, to move forward you can use tab key and to move backward you can use shift+tab key.

**Comment by anuvenkatesh1** [comment]
@Meghana-MSFT 
To meet Accessibility standards for Keyboard users, all interactive elements must be accessible via both tab and shift+tab keys
Issue encountered here is the close button is not accessible via tab navigation but only via shift+tab (reverse navigation) which is violating accessibility standards.
Please check https://www.w3.org/WAI/WCAG21/Understanding/no-keyboard-trap (A modal dialog box under Examples) which states "tabbing from the last control in the dialog takes focus to the first control in the dialog". In this case, when we tab from the last control of the dialog, focus doesn't go to the first control of the dialog which is the close button.
You can check with any modal dialog control and observe that tabbing in the forward direction always loops through all the interactive elements within the dialog and there isn't any expectation that if a control is accessible via shift+tab alone, it can be considered accessibility compliant. 

Thank you

**Comment by Meghana-MSFT** [comment]
@qjkobe, @anuvenkatesh1 - We raised a bug for this, we will keep you posted on the updates. Thank you.

**Comment by Prasad-MSFT** [comment]
Hi @qjkobe, I just came across this thread. Is this issue still reproable?

**Comment by qjkobe** [comment]
Hi @Prasad-MSFT ,

I re-tested this issue and this issue was fixed.

Best Regards,
Kris
